File: share-code/bai-tap-lap-trinh/python/condition/49.py
your_number = int(input("Nhập một năm: "))

notice = "Không là năm nhuận"

if (your_number % 400 == 0) or (your_number % 4 == 0 and your_number % 100 != 0):
    notice = "Năm nhuận"

print(notice)    

File: share-code/bai-tap-lap-trinh/cpp/49.cpp
#include <iostream>
using namespace std;
#include <string>

int main()
{
    int year = 2000;
    string notice = "Không là năm nhuận ";
    if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
        notice = "Năm nhuận";
    }
    cout << notice;
}